src/transformers/models/nezha/configuration_nezha.py

```diff
@@ -57,15 +57,14 @@ class NezhaConfig(PretrainedConfig):
     >>> # Initializing an Nezha configuration
     >>> configuration = NezhaConfig()
 
-    >>> # Initializing a model from the Nezha-base style configuration model
+    >>> # Initializing a model (with random weights) from the Nezha-base style configuration model
     >>> model = NezhaModel(configuration)
 
     >>> # Accessing the model configuration
     >>> configuration = model.config
     ```"""
-
-    pretrained_config_archive_map = NEZHA_PRETRAINED_CONFIG_ARCHIVE_MAP
     model_type = "nezha"
+    pretrained_config_archive_map = NEZHA_PRETRAINED_CONFIG_ARCHIVE_MAP
```

Comment:
Let's not change this :-). You can remove the empty line though. Thanks

Reply:
@ydshieh Thank you! I have changed it back. Would you please review it again? :)